js1/funkce-arg-udalosti/zpozdeni-casovace.md

```diff
@@ -0,0 +1,33 @@
+## Zpoždění a časovače
+
+Dobrá zpráva pro začínající programátory je, že psát funkce očekávající jiné funkce na vstupu je spíše pokročilá věc a budeme ji používat až při práci v Reactu. Zatím budeme funkce tohoto ražení používat už hotové, zabuduované přímo v JavaScriptu.
+
+Jedna ze situací, kdy se nám velmi hodí použít funkci jako argument, nastává, když chceme v JavaScriptu provedení nějaké funkce pozdržet nebo její volání pravidelně opakovat. Vzpomeňte si na příklad [kvíz](../dom-innerhtml/cv-manipulace#cvlekce%3Ekviz). Představme si, že chceme, aby uživatel měl na každou odpověď nějaký časový limit. Po zobrazení otázky chceme počkat 5 vteřin a poté vypsat něco jako „čas vypršel“. Vyrobíme si tedy funkci, která vypisuje naši zprávu, zatím pro jednoduchost přimo do `body`.
```

Comment:
Za mě doporučuji se odkazům v materiálech vyhýbat, protože je to něco, co se často rozbíjí. Je velmi těžké při přejmenovávání a jiném přesouvání dohlédnout na tyto slabé reference v podobně odkazů.

Pokud by kodim časem mělo nějaký nástroj, který by mimo jiné na CI mohl dělat validaci, bylo by to fajn.

Reply:
Odkaz jsem raději odstranil